Makefile

```
@@ -472,6 +472,15 @@ lint: $(GOLANGCI_LINT_BIN)
 update-tekton:
 	SKIP_UPDATE=false ./update-tekton.sh .tekton/*.yaml
 
+.PHONY: validate-commits validate-commits-range
```

checkmake (0.2.2)

[warning] Missing required phony target "clean"

(minphony)